css/navigation.css

```diff
@@ -1 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
 
+nav {
+  display: flex;
```

Comment:
This will not affect the `<li>` tags, because they are not **direct children** of the nav. To create a horizontal navbar, use `display: flex;` on the `ul`. You can select the `ul` with `nav ul` or give it a class like `navigation__ul`.